lib/scss_lint/cli.rb

```diff
@@ -169,6 +172,15 @@ def report_lints(options, lints, files)
       end
     end
 
+    def auto_correct(engines)
+      runner.lints.each do |lint|
+        lint.auto_correct.call if lint.auto_correct
+      end
+      engines.each do |engine|
+        puts engine.tree.to_scss
```

Comment:
I don't think this works the way you think it does. Calling `to_scss` is likely to change more of the original source code than you originally intended due to the quirks of the Sass engine. 

Furthermore, using `puts` to output the results prevents us from controlling the file we output to via p[`--out` flags](https://github.com/brigade/scss-lint/pull/273). A better solution would simply update the file at the end of the run.

However, the big show-stopper here is using `to_scss`, which will butcher the original source tree.